summing-numbers-3.php

```diff
@@ -0,0 +1,10 @@
+<?php
+$sum = 0;
+for ($i=2; $i<9; $i++){
+    $sum+=$i;
+    if ($sum==35){
+        echo "I love you more than everything <br />";
```

Comment:
@Katrin12323 I am the happiest man. I love you too my sunshine girl :))) :1st_place_medal: